Index: src/main/java/edu/bsu/cs/basicInfo/BasicInfoEngine.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/edu/bsu/cs/basicInfo/BasicInfoEngine.java b/src/main/java/edu/bsu/cs/basicinfo/BasicInfoEngine.java
rename from src/main/java/edu/bsu/cs/basicInfo/BasicInfoEngine.java
rename to src/main/java/edu/bsu/cs/basicinfo/BasicInfoEngine.java
--- a/src/main/java/edu/bsu/cs/basicInfo/BasicInfoEngine.java	
+++ b/src/main/java/edu/bsu/cs/basicinfo/BasicInfoEngine.java	
@@ -1,5 +1,7 @@
 package edu.bsu.cs.basicinfo;
 
+import javafx.scene.layout.VBox;
+
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.List;
